mrobosub_hal/mrobosub_hal/thruster_controller.py

```diff
@@ -100,7 +109,7 @@ def convert_pwm_signal(self, pwm_raw: float) -> Optional[int]:
 
     # in case of invalid PWM or motor number parameters, does not send any updated signal to the motor controller
     def send_signal(self, motor: int, pwm_raw: float) -> int:
-        if getattr(self.srv.config, f"motor{motor}_rev"):
+        if self.get_parameter("thruster_reverse").get_parameter_value().bool_array_value[motor]:  # type: ignore
```

Comment:
Fuck ros 2 bruh this is disgusting

Reply:
It is actually awful, I spent over an hour doing it multiple different ways and this is by far the least bad